aws-lambda/massdriver.yaml

```diff
@@ -57,6 +59,7 @@ params:
     region:
       title: AWS Region
       $ref: https://raw.githubusercontent.com/massdriver-cloud/artifact-definitions/main/definitions/types/aws-region.json
+      $md.immutable: true
```

Comment:
Added immutable here and for ECR URI because I'm assuming they're destructive changes as with most other bundles